tasks/setup.yml

```diff
@@ -82,3 +95,21 @@
   stat:
     path: "{{ current_venv_dir }}"
   register: venv_check_results
+
+- name: Define dynamic run tmpfile(s)
+  lineinfile:
+    dest: /etc/tmpfiles.d/{{ item[0] }}-{{ item[1] }}.conf
+    line: "d /run/{{item[0]}}-{{item[1]}} 0755 {{django_stack_gcorn_user}} {{django_stack_gcorn_group}} -"
+    create: yes
+  with_nested:
+    - ['gunicorn']
+    - "{{ django_stack_gcorn_ports.keys() }}"
+
+- name: Define static run tmpfile(s)
+  lineinfile:
+    dest: /etc/tmpfiles.d/{{ item[0] }}
+    line: "d {{item[1]}}/{{item[0]}} 0755 {{django_stack_gcorn_user}} {{django_stack_gcorn_group}} -"
+    create: yes
+  with_nested:
+    - ['celery']
+    - ['/run/', '/var/log']
```

Comment:
Why use `with_nested` on these tasks? `item[0]` resolves to a single string each time, so why not just hardcode it? Then you can use regular ol' `with_items` and the task becomes a lot more readable. Am I missing something?

Reply:
I was trying to abstract this stuff in case we need to append to it later.. but yeah it just looks kind of silly